packages/@webex/plugin-cc/src/features/constants.ts

```diff
@@ -1,4 +1,7 @@
 // making query params configurable for List Teams and List Aux Codes API
 export const DEFAULT_PAGE = 0;
 export const DEFAULT_PAGE_SIZE = 10;
-export const DEFAULT_ATTRIBUTES = ['id', 'name'];
+export const DEFAULT_ATTRIBUTES = [];
+export const AGENT_STATE_AVAILABLE_ID = '0';
```

Comment:
For 'Available' state the auxCodeId will be '0' this is as per the Agentx Desktop.